kubera/dmaas/credentials/minio_test.go

```diff
@@ -0,0 +1,44 @@
+package minio
+
+import (
+	"testing"
+
+	auth "github.com/mayadata-io/kubera-api-testing/kubera/authentication"
+	"github.com/mayadata-io/kubera-api-testing/kubera/connection"
+)
+
+func TestMinioCredentialUpdate(t *testing.T) {
+	credentialTitle := "minio-cred-demo"
```

Comment:
This function represents the brain / logic of adding a minio credential.
Hence this should not be inside _test.go file.
_test.go file needs to provide various combinations to test adding minio credential.